### ISSUE TITLE
Add new  models in the Model Selection section

### ISSUE THREAD

**Issue by mllogeo** [rmozart/North-Sea-Electrofacies-Clustering#15]
XGBoost, CatBoost, LightGBM, and Gradient Boosted Trees

**Comment by rmozart** [comment]
All models, except Gradient Boosted Trees, were added and are already on the remote repository.
